modules/src/ics02_client/msgs.rs

```diff
@@ -92,7 +92,7 @@ impl TryFrom<RawMsgCreateClient> for MsgCreateAnyClient {
             .ok_or_else(|| Kind::DecodeMessage.context(error::Kind::InvalidRawClientState))?;
 
         let client_type = match raw_client_state.type_url.as_str() {
-            "/ibc.tendermint.ClientState" => Ok(ClientType::Tendermint),
+            "/ibc.lightclients.tendermint.v1.ClientState" => Ok(ClientType::Tendermint),
```

Comment:
Can we extract this string as a constant as part of this PR already?

Reply:
Oops...forgot this one. Actually I had this on my mind for a while, do we still need the `client_type` in `MsgCreateAnyClient`? We do the same match when we decide on the enum type for `AnyClientState` which is included there. And looking at the code we only initialize this field.

Reply:
Good point! It seems we can indeed remove it and replace it with a function which does the match you suggest.

Reply:
By the way, we should actually enforce in the TryFrom instance that the `client_state` we are getting in the raw message matches the type of client state, even if we don't store, but for ensuring that the raw message we are getting is consistent.